<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoGra2</class>
 <widget class="QMainWindow" name="OknoGra2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="maszyna"/>
   </widget>
   <widget class="QPushButton" name="SPINPRZYCISK">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>SPINEK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="JapkoEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BananEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="WinogronkoEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="WisniaEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BonusEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kwotaWygranejLineEdit_Japko">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kwotaWygranejLineEdit_Banan&#10;">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kwotaWygranejLineEdit_Winogronko&#10;">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kwotaWygranejLineEdit_Wisnia">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mno≈ºnik</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Szanse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>üçé</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>üçå</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>üçá</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>üçí</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>üéÅ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kwotaWygranejLineEdit_Bonus">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mno≈ºnik na
 bonusie</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="saldoLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Saldo: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_1_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_2_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_3_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_4_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_8_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_6_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_7_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_5_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_12_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_10_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_14_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_16_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 16</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_13_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 13</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_11_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_15_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 15</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_9_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="l_17_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia 17</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mnoznikBonusuLineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WygraneLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Wygrane</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="LacznaWygrana_Label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>≈ÅƒÖczna Wygrana</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AUTOSPINPRZYCISK">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AUTOSPIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba
 FreeSpin√≥w</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LiczbaFreeSpinowLineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LiczbaDodatkowychFreeSpinowLineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodatkowe
 FreeSpiny</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Freespiny_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FreeSpiny: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="stawkaComboBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Stawka_Label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stawka:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powrotButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Powr√≥t</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
